mlrun/platforms/other.py

```diff
@@ -23,7 +23,7 @@
 from mlrun.errors import MLRunInvalidArgumentError
 from mlrun.utils.helpers import logger
 
-from .iguazio import mount_v3io
+from ..pipelines.iguazio import mount_v3io
```

Comment:
Maybe I'm missing something - this file is full of modifiers that use `container_op` for their purposes and reference `kfp.dsl`. Shouldn't they also be moved to the `pipelines` directory like you did for `mount_v3io` and its friends above? Or did you just miss this?

Reply:
These absolutely will be moved. I didn't get around to doing it yet.